app/routes/wiki-page.js

```diff
@@ -155,6 +156,13 @@ export default Route.extend(
 								adsModule.init(model.adsContext);
 							}
 						});
+
+						// If it's an article page and the extension is enabled, load the Feeds & Posts module
+						if (model.type === 'article' && this.get('wikiVariables.enableFeedsAndPosts')) {
+							feedsAndPosts.getModule().then((fpModule) => {
```

Comment:
I was told that the name of the project is feeds

Reply:
if it is enabled will this be visible on all pages?

Reply:
The "in-development" name is Feeds and Posts, but maybe we'll rename it to something shorter later on.

Product wants it integrated on any article page that isn't the main page.